tests/typecheck/models/test_abstract.yml

```diff
@@ -33,3 +33,28 @@
 
             class MyModel(BaseModel):
                 field = models.IntegerField()
+
+- case: test_can_instantiate_with_recursive_relation_on_abstract_model
+  main: |
+    from myapp.models import Concrete, Recursive
+    first = Concrete.objects.create(parent=None)
+    Concrete.objects.create(parent=first)
+    Recursive.objects.create(parent=None)
+    Recursive(parent=Recursive(parent=None))  # E: Unexpected attribute "parent" for model "Recursive"
```

Comment:
For the curious one; it was initialisation of an abstract model(i.e. this line) that raised the error seen in the provided traceback